cnf-tests/hack/govet.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+. $(dirname "$0")/common.sh
+echo "INFO: Running govet on cnf-tests/testsuites/..."
+go vet ./testsuites/...
```

Comment:
that's missing the `GO111MODULE` disabling (??)

Reply:
It is, but I wasn't even sure why that was there in the first place.

This seems to run fine.

Reply:
Also the old way seemed broken, as you noticed.

Reply:
Copying the comment from the other PR, why not sharing a single govet.sh file in the root `hack` folder?